Update pyyaml version

javanlacerda opened the pull request:
Closes #70230
It updates the  PyYaml version from 5.1 to 6. The 5.1 version has security vulnerabilities as described in these links:
* https://vulners.com/github/GHSA-3PQX-4FQF-J49F
* https://vulners.com/osv/OSV:PYSEC-2020-96
* https://vulners.com/osv/OSV:PYSEC-2021-142

kartben commented:
> Hey @galak, I changed the commit for being compliant with checks, should I do something to trigger the job?

As a first-time contributor, you just need to wait for one of the project's collaborators to trigger it, which I just did. Thanks!

javanlacerda commented:
> > Hey @galak, I changed the commit for being compliant with checks, should I do something to trigger the job?
> 
> As a first-time contributor, you just need to wait for one of the project's collaborators to trigger it, which I just did. Thanks!

Thank you very much @kartben, I appreciate that!!

kartben commented:
> hey @kartben, is there any way for testing the commits message compliance locally?

there is!

```
~/zephyrproject/zephyr/scripts/ci/check_compliance.py
```

javanlacerda committed:
scripts: dts: update pyyaml version

The currently used PyYaml version has some vulnerabilies as
described on the pull request description. It updates to
version 6.0, removing these supply chain vulnerabily.
The OSSF Scorecard was the tool used for discovering
 these vulnerabilties.

Signed-off-by: Javan lacerda <javanlacerda@google.com>

javanlacerda commented:
hey @kartben, I appreciate your help! I think it is compliant now.

javanlacerda commented:
I don't understand why the `Documentation Build` is failing :/

kartben commented:
> I don't understand why the `Documentation Build` is failing :/

I re-ran the job -- random issue with the Github runner is my guess.

javanlacerda commented:
> > I don't understand why the `Documentation Build` is failing :/
> 
> I re-ran the job -- random issue with the Github runner is my guess.

Awesome! Thank you very much!

kartben commented:
@aescolar @nashif as release managers, please consider re-assigning this one to either of you and merging. Thanks!
